### DISCUSSION TITLE
Thoughts on redesign of "Create Request" interface?

### DISCUSSION THREAD

**Discussion by bdaley** [sourceryapp/sourcery-web-app#215] [Ideas]
Not sure this will "cut it" for the launch of the institutional version. Maybe we do cards with photos? We'll be launching with only a couple of institutions. UConn is likely to be one of them.

cc: @bfoti @mueller-alex 

![image](https://user-images.githubusercontent.com/382814/114248972-35e7d880-9967-11eb-82df-16f79ae2a8e9.png)

**Comment by bfoti** [comment]
Cards with photos would work well  for now...I'll see if Wes can get Rebecca, Dan, and the Folger contact to send over a photo of each respective building. I think it would be good to establish that as part of our process/pipeline of getting partners added to our system; I could see us using the images elsewhere later on when/if we move past using them here

**Comment by bdaley** [comment]
Moving any further discussion to the newly created issue: https://github.com/GreenhouseStudios/sourcery-web-app/issues/218
